### PR TITLE
Create RubyChallenge Model and migration

### PR DESCRIPTION
**Pull request by Jplferreyra**
Please review this

### CODE REVIEW

File: app/__init__.py [hunk 1]
```diff
@@ -1,36 +1,37 @@
...
+
+def create_app(config_name='development'):
+    app = Flask(__name__)
+
+    app.config.from_object(config[config_name])
+    app.config['SQLALCHEMY_TRACK_MODIFICATIONS'] = False
+    config[config_name].init_app(app)
+
+    from .models import User, Role
+    from .ruby.models import RubyChallenge
```
Comment (Gonzalo-FloresM) [inline app/__init__.py:17]: Why did you add this line mr?

File: app/ruby/models.py [hunk 2]
```diff
@@ -0,0 +1,10 @@
+from .. import db
+
+class RubyChallenge(db.Model):
+
```
[the remark this answers is not in the document]
Comment (Jplferreyra) [reply app/ruby/models.py:4]: Dont cry

File: app/__init__.py (hunk 1 above)
Comment (Jplferreyra) [reply app/__init__.py:17]: So the model is visible for alembic
